lib/browser/guest-view-manager.js

```diff
@@ -128,17 +128,6 @@ const createGuest = function (embedder, params) {
     }
   })
 
-  // Forward internal web contents event to embedder to handle
-  // native window.open setup
-  guest.on('-add-new-contents', (...args) => {
```

Comment:
I thought this was needed to make native `window.open` work in webview, is it no longer needed?

Reply:
It's no longer needed - I think this was there to make sure that the `-add-new-contents` is handled because only `BrowserWindow`s and `BrowserView`s had this event handled, but I expanded that to be handled for every `WebContents` that's not `remote`.
https://github.com/electron/electron/pull/19703/files#diff-940e61f1a69c6bbbeb31968b6fc49d1cR378